### PR TITLE
Added point of contact (contact person) to Dream

### PR DESCRIPTION
**Pull request by basels**
visible to dream-creator/admin/guide, editable by admin/guide

### CODE REVIEW

**Comment by basels** [comment]
https://github.com/Midburn/dreams/issues/111

File: app/views/camps/show.html.erb [hunk 1]
```diff
@@ -371,6 +371,19 @@
                     <div class="panel-heading"><h2 class="header-sub-heading"><%=t :creation_info_headline %></h2></div>
                 <div class="panel-body">
 
+                <% if @camp.active? %>
```
Comment (rootux) [inline app/views/camps/show.html.erb]: How does the fact the the camp is active related here?

Comment (rootux) [reply app/views/camps/show.html.erb]: Also - can you make it into a feature that can be turned on / off (in sake of borderland and others).
(add it as a setting to firestarter_settings.xml file with ability to update it using an ENV var and default - turn it off.

**Review by rootux — CHANGES_REQUESTED** [review]
Looks great-  added 2 comments.

File: app/views/camps/show.html.erb (hunk 1 above)
Comment (basels) [reply app/views/camps/show.html.erb]: Fixed

**Pull request merged**